src/components/application_manager/test/commands/mobile/set_global_properties_test.cc

```diff
@@ -90,6 +94,8 @@ class SetGlobalPropertiesRequestTest
   }
 
   ~SetGlobalPropertiesRequestTest() {
+    // Required for proper DataAccessor release
+    Mock::VerifyAndClearExpectations(&app_mngr_);
```

Comment:
@LuxoftAKutsan Let me try to put some light on this one since it has a slightly different meaning in this case: this call prevents potential crash while destructing owned lock (see [set_global_properties_test.cc](https://github.com/smartdevicelink/sdl_core/pull/1378/files#diff-33ed62bff3873bac3761ec6756e492fe):474) by forcing mock expectations cleanup at the earliest stage.